code/modules/paperwork/carbonpaper.dm

```diff
@@ -39,7 +45,7 @@
 		user.put_in_hands(Copy)
 
 /obj/item/paper/carbon/attack_hand(mob/living/user, list/modifiers)
```

Comment:
Change this to `/attack_hand_secondary`, perform your behavior, then return CANCEL_ATTACK_CHAIN.

Reply:
Or rather, add `/attack_hand_secondary` instead of using modifiers.